packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -438,7 +440,16 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
         child: AutocompleteHighlightedOption(
           highlightIndexNotifier: _highlightedOptionIndex,
           child: Builder(
-            builder: (BuildContext context) => widget.optionsViewBuilder(context, _select, _options),
+            builder: (BuildContext context) {
+              return UnconstrainedBox(
+                constrainedAxis: Axis.vertical,
+                alignment: Alignment.topLeft,
+                child: SizedBox(
+                  width: _fieldBoxConstraints.maxWidth,
+                  child: widget.optionsViewBuilder(context, _select, _options),
```

Comment:
Question: Should I add a flag for this, or even pass the Size to the optionsViewBuilder and make users include the UnconstrainedBox and SizedBox? I figure most users do want this width-matching behavior, which is why I made it mandatory, but it could be a breaking change for some...

Reply:
Thanks for the analysis, I'm on board with this. I'll leave this as-is but add a TODO with a link to this thread if/when someone starts asking for it.